src/efftester.ml

```diff
@@ -740,6 +793,7 @@ module GeneratorsWithContext (Ctx : Context) = struct
     | Float -> Gen.map (fun f -> LitFloat f) float_gen
     | Bool -> Gen.map (fun b -> LitBool b) Gen.bool
     | String -> Gen.map (fun s -> LitStr s) string_gen
+    | Option _ -> failwith "literal_gen: option arg. should not happen"
```

Comment:
I'm not completely sure what `literal_gen` is used for, but I would note that `List` is supported below, and it should be fairly easy to support `Option` using the same logic. (A literal at `int option` could be `None` or `Some 3`, for example).